components/admin/lessons/AdminLessonSideNav.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import { ApolloError } from '@apollo/client'
+import { AlertFillIcon, ChevronRightIcon } from '@primer/octicons-react'
+import { isNumber } from 'lodash'
+import { Spinner } from 'react-bootstrap'
+import { get } from 'lodash'
+import styles from '../../../scss/adminLessonSideNav.module.scss'
+
+type Item = {
+  id: number
+  name: string
+  content: string
+  lesson: { id?: number }
+}
+
+type Items = Item[]
+
+type Props = {
+  items: Items | undefined
+  title: string
+  lessonId: number
+  onAddItem: (m: Item | undefined) => void
+  loading?: boolean
+  error?: ApolloError
+  active?: number
+  onClick: (item: Item) => void
+}
+
+const AdminLessonSideNav = ({
+  items,
+  onAddItem,
+  loading,
+  error,
+  active,
+  title,
+  lessonId,
+  onClick
```

Comment:
more of a nitpick here,`onSelect` is a better name here, because on mobile devices there's no click. And also for accessibility there's no click either most ppl use keyboard tab select.